src/lib.rs

```diff
@@ -78,6 +78,7 @@
 //! let output = command.wait().expect("Couldn't get cargo's exit status");
 //! ```
 
+use camino::Utf8PathBuf;
```

Comment:
Please publicly reexport `camino` so that crates depending on `cargo_metadata` don't have to add a dependency on camino themselves (and potentially run into versioning issues in the future).

Reply:
Done, thanks.